apps/ui/src/components/mock/applicationMockData.ts

```diff
@@ -24,6 +24,118 @@ import { ApplicationStates } from '@pcgl-daco/data-model/src/types';
 
+export const mockTableData = [
+	{
+		id: 1,
+		institution: 'Ontario Institute For Cancer Research (OICR)',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime() + 10000000,
+		applicant_institutional_email: 'testUser@oicr.on.ca',
+		state: 'DAC_REVIEW',
+	},
+
+	{
+		id: 2,
+		institution: 'University of Guelph',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@uoguelph.ca',
+		state: 'DAC_REVIEW',
+	},
+	{
+		id: 5,
+		institution: 'Unknown Institution U',
+		institution_country: 'United States',
+		applicant_full_name: 'Mysterious User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'mysterious@example.com',
+		state: 'REJECTED',
+	},
+	{
+		id: 3,
+		institution: 'University of Toronto',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@utoronto.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 4,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'DAC_REVIEW',
+	},
+	{
+		id: 5,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 6,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 7,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 8,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 9,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 10,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
+		state: 'APPROVED',
+	},
+	{
+		id: 11,
+		institution: 'McGill University',
+		institution_country: 'Canada',
+		applicant_full_name: 'Test User',
+		updated_at: new Date().getTime(),
+		applicant_institutional_email: 'testUser@mcgill.ca',
```

Comment:
unlikely to cause issues but always good rule of thumb to use the `example.com` domain for emails. won't accidentally get sent somewhere its not meant to